compilation using conda development environment seems to be broken

I ve tried to create a conda development environment following the usual steps of the documentation, but I m getting compilation errors
```
Scanning dependencies of target nnpdf
[  2%] Building CXX object libnnpdf/CMakeFiles/nnpdf.dir/src/common.cc.o
[  4%] Building CXX object libnnpdf/CMakeFiles/nnpdf.dir/src/commondata.cc.o
/home/tommy/physics/nnpdfgit/nnpdf/libnnpdf/src/commondata.cc: In copy constructor 'NNPDF::CommonData::CommonData(const NNPDF::CommonData&)':
/home/tommy/physics/nnpdfgit/nnpdf/libnnpdf/src/commondata.cc:235:35: warning: implicitly-declared 'NNPDF::sysError& NNPDF::sysError::operator=(const NNPDF::sysError&)' is deprecated [-Wdeprecated-copy]
  235 |         fSys[i][l] = set.fSys[i][l];
      |                                   ^
In file included from /home/tommy/physics/nnpdfgit/nnpdf/libnnpdf/src/commondata.cc:24:
/home/tommy/physics/nnpdfgit/nnpdf/libnnpdf/src/NNPDF/commondata.h:122:5: note: because 'NNPDF::sysError' has user-provided 'NNPDF::sysError::sysError(const NNPDF::sysError&)'
  122 |     sysError(sysError const& o):
      |     ^~~~~~~~
/home/tommy/physics/nnpdfgit/nnpdf/libnnpdf/src/commondata.cc: In constructor 'NNPDF::CommonData::CommonData(const NNPDF::CommonData&, const std::vector<int>&)':
/home/tommy/physics/nnpdfgit/nnpdf/libnnpdf/src/commondata.cc:312:41: warning: implicitly-declared 'NNPDF::sysError& NNPDF::sysError::operator=(const NNPDF::sysError&)' is deprecated [-Wdeprecated-copy]
  312 |         fSys[i][l] = set.fSys[mask[i]][l];
      |                                         ^
In file included from /home/tommy/physics/nnpdfgit/nnpdf/libnnpdf/src/commondata.cc:24:
/home/tommy/physics/nnpdfgit/nnpdf/libnnpdf/src/NNPDF/commondata.h:122:5: note: because 'NNPDF::sysError' has user-provided 'NNPDF::sysError::sysError(const NNPDF::sysError&)'
  122 |     sysError(sysError const& o):
      |     ^~~~~~~~
[  6%] Building CXX object libnnpdf/CMakeFiles/nnpdf.dir/src/chisquared.cc.o
[  8%] Building CXX object libnnpdf/CMakeFiles/nnpdf.dir/src/dataset.cc.o
[ 11%] Building CXX object libnnpdf/CMakeFiles/nnpdf.dir/src/experiments.cc.o
[ 13%] Building CXX object libnnpdf/CMakeFiles/nnpdf.dir/src/fastkernel.cc.o
/home/tommy/physics/nnpdfgit/nnpdf/libnnpdf/src/fastkernel.cc: In constructor 'NNPDF::FKTable::FKTable(const string&, const std::vector<std::__cxx11::basic_string<char> >&)':
/home/tommy/physics/nnpdfgit/nnpdf/libnnpdf/src/fastkernel.cc:306:28: warning: implicitly-declared 'NNPDF::FKHeader& NNPDF::FKHeader::operator=(const NNPDF::FKHeader&)' is deprecated [-Wdeprecated-copy]
  306 |     fFKHeader = FKHeader(is);
      |                            ^
/home/tommy/physics/nnpdfgit/nnpdf/libnnpdf/src/fastkernel.cc:62:3: note: because 'NNPDF::FKHeader' has user-provided 'NNPDF::FKHeader::FKHeader(const NNPDF::FKHeader&)'
   62 |   FKHeader::FKHeader(FKHeader const& ref):
      |   ^~~~~~~~
[ 15%] Building CXX object libnnpdf/CMakeFiles/nnpdf.dir/src/fkgenerator.cc.o
[ 17%] Building CXX object libnnpdf/CMakeFiles/nnpdf.dir/src/fkset.cc.o
[ 20%] Building CXX object libnnpdf/CMakeFiles/nnpdf.dir/src/lhapdfset.cc.o
[ 22%] Building CXX object libnnpdf/CMakeFiles/nnpdf.dir/src/logger.cc.o
[ 24%] Building CXX object libnnpdf/CMakeFiles/nnpdf.dir/src/nnmpi.cc.o
[ 26%] Building CXX object libnnpdf/CMakeFiles/nnpdf.dir/src/nnpdfdb.cc.o
[ 28%] Building CXX object libnnpdf/CMakeFiles/nnpdf.dir/src/parametrisation.cc.o
[ 31%] Building CXX object libnnpdf/CMakeFiles/nnpdf.dir/src/pdfset.cc.o
[ 33%] Building CXX object libnnpdf/CMakeFiles/nnpdf.dir/src/positivity.cc.o
[ 35%] Building CXX object libnnpdf/CMakeFiles/nnpdf.dir/src/randomgenerator.cc.o
[ 37%] Building CXX object libnnpdf/CMakeFiles/nnpdf.dir/src/thpredictions.cc.o
/home/tommy/physics/nnpdfgit/nnpdf/libnnpdf/src/thpredictions.cc:398: warning: ignoring #pragma omp parallel [-Wunknown-pragmas]
  398 |   #pragma omp parallel for
      | 
/home/tommy/physics/nnpdfgit/nnpdf/libnnpdf/src/thpredictions.cc:448: warning: ignoring #pragma omp parallel [-Wunknown-pragmas]
  448 |   #pragma omp parallel for
      | 
[ 40%] Building CXX object libnnpdf/CMakeFiles/nnpdf.dir/src/utils.cc.o
[ 42%] Building CXX object libnnpdf/CMakeFiles/nnpdf.dir/src/pathlib.cc.o
[ 44%] Linking CXX shared library libnnpdf.so
[ 44%] Built target nnpdf
Scanning dependencies of target nnpdf_swig_compilation
[ 46%] Swig compile ./src/nnpdf.i for python
[ 46%] Built target nnpdf_swig_compilation
Scanning dependencies of target _nnpdf
[ 48%] Building CXX object libnnpdf/wrapper/CMakeFiles/_nnpdf.dir/CMakeFiles/_nnpdf.dir/nnpdfPYTHON_wrap.cxx.o
/home/tommy/physics/nnpdfgit/nnpdf/bld/libnnpdf/wrapper/CMakeFiles/_nnpdf.dir/nnpdfPYTHON_wrap.cxx:1843:36: warning: cast between incompatible function types from 'PyObject* (*)(PyObject*)' {aka '_object* (*)(_object*)'} to 'PyCFunction' {aka '_object* (*)(_object*, _object*)'} [-Wcast-function-type]
 1843 |   {(char *)"disown",  (PyCFunction)SwigPyObject_disown,  METH_NOARGS,  (char *)"releases ownership of the pointer"},
      |                                    ^~~~~~~~~~~~~~~~~~~
/home/tommy/physics/nnpdfgit/nnpdf/bld/libnnpdf/wrapper/CMakeFiles/_nnpdf.dir/nnpdfPYTHON_wrap.cxx:1844:36: warning: cast between incompatible function types from 'PyObject* (*)(PyObject*)' {aka '_object* (*)(_object*)'} to 'PyCFunction' {aka '_object* (*)(_object*, _object*)'} [-Wcast-function-type]
 1844 |   {(char *)"acquire", (PyCFunction)SwigPyObject_acquire, METH_NOARGS,  (char *)"acquires ownership of the pointer"},
      |                                    ^~~~~~~~~~~~~~~~~~~~
/home/tommy/physics/nnpdfgit/nnpdf/bld/libnnpdf/wrapper/CMakeFiles/_nnpdf.dir/nnpdfPYTHON_wrap.cxx:1847:36: warning: cast between incompatible function types from 'PyObject* (*)(PyObject*)' {aka '_object* (*)(_object*)'} to 'PyCFunction' {aka '_object* (*)(_object*, _object*)'} [-Wcast-function-type]
 1847 |   {(char *)"next",    (PyCFunction)SwigPyObject_next,    METH_NOARGS,  (char *)"returns the next 'this' object"},
      |                                    ^~~~~~~~~~~~~~~~~
/home/tommy/physics/nnpdfgit/nnpdf/bld/libnnpdf/wrapper/CMakeFiles/_nnpdf.dir/nnpdfPYTHON_wrap.cxx:1848:36: warning: cast between incompatible function types from 'PyObject* (*)(SwigPyObject*)' {aka '_object* (*)(SwigPyObject*)'} to 'PyCFunction' {aka '_object* (*)(_object*, _object*)'} [-Wcast-function-type]
 1848 |   {(char *)"__repr__",(PyCFunction)SwigPyObject_repr,    METH_NOARGS,  (char *)"returns object representation"},
      |                                    ^~~~~~~~~~~~~~~~~
/home/tommy/physics/nnpdfgit/nnpdf/bld/libnnpdf/wrapper/CMakeFiles/_nnpdf.dir/nnpdfPYTHON_wrap.cxx: In function 'PyTypeObject* SwigPyObject_TypeOnce()':
/home/tommy/physics/nnpdfgit/nnpdf/bld/libnnpdf/wrapper/CMakeFiles/_nnpdf.dir/nnpdfPYTHON_wrap.cxx:2007:5: warning: missing initializer for member '_typeobject::tp_vectorcall' [-Wmissing-field-initializers]
 2007 |     };
      |     ^
/home/tommy/physics/nnpdfgit/nnpdf/bld/libnnpdf/wrapper/CMakeFiles/_nnpdf.dir/nnpdfPYTHON_wrap.cxx:2007:5: warning: missing initializer for member '_typeobject::tp_print' [-Wmissing-field-initializers]
/home/tommy/physics/nnpdfgit/nnpdf/bld/libnnpdf/wrapper/CMakeFiles/_nnpdf.dir/nnpdfPYTHON_wrap.cxx: In function 'PyTypeObject* SwigPyPacked_TypeOnce()':
/home/tommy/physics/nnpdfgit/nnpdf/bld/libnnpdf/wrapper/CMakeFiles/_nnpdf.dir/nnpdfPYTHON_wrap.cxx:2195:5: warning: missing initializer for member '_typeobject::tp_vectorcall' [-Wmissing-field-initializers]
 2195 |     };
      |     ^
/home/tommy/physics/nnpdfgit/nnpdf/bld/libnnpdf/wrapper/CMakeFiles/_nnpdf.dir/nnpdfPYTHON_wrap.cxx:2195:5: warning: missing initializer for member '_typeobject::tp_print' [-Wmissing-field-initializers]
/home/tommy/physics/nnpdfgit/nnpdf/bld/libnnpdf/wrapper/CMakeFiles/_nnpdf.dir/nnpdfPYTHON_wrap.cxx: In function 'PyTypeObject* swig_varlink_type()':
/home/tommy/physics/nnpdfgit/nnpdf/bld/libnnpdf/wrapper/CMakeFiles/_nnpdf.dir/nnpdfPYTHON_wrap.cxx:63488:7: warning: missing initializer for member '_typeobject::tp_vectorcall' [-Wmissing-field-initializers]
63488 |       };
      |       ^
/home/tommy/physics/nnpdfgit/nnpdf/bld/libnnpdf/wrapper/CMakeFiles/_nnpdf.dir/nnpdfPYTHON_wrap.cxx:63488:7: warning: missing initializer for member '_typeobject::tp_print' [-Wmissing-field-initializers]
/home/tommy/physics/nnpdfgit/nnpdf/bld/libnnpdf/wrapper/CMakeFiles/_nnpdf.dir/nnpdfPYTHON_wrap.cxx: In instantiation of 'static Type swig::traits_as<Type, swig::pointer_category>::as(PyObject*, bool) [with Type = NNPDF::DataSet; PyObject = _object]':
/home/tommy/physics/nnpdfgit/nnpdf/bld/libnnpdf/wrapper/CMakeFiles/_nnpdf.dir/nnpdfPYTHON_wrap.cxx:3962:64:   required from 'Type swig::as(PyObject*, bool) [with Type = NNPDF::DataSet; PyObject = _object]'
/home/tommy/physics/nnpdfgit/nnpdf/bld/libnnpdf/wrapper/CMakeFiles/_nnpdf.dir/nnpdfPYTHON_wrap.cxx:4525:20:   required from 'swig::SwigPySequence_Ref<T>::operator T() const [with T = NNPDF::DataSet]'
/home/tommy/physics/nnpdfgit/nnpdf/bld/libnnpdf/wrapper/CMakeFiles/_nnpdf.dir/nnpdfPYTHON_wrap.cxx:4984:30:   required from 'void swig::assign(const SwigPySeq&, Seq*) [with SwigPySeq = swig::SwigPySequence_Cont<NNPDF::DataSet>; Seq = std::vector<NNPDF::DataSet>]'
/home/tommy/physics/nnpdfgit/nnpdf/bld/libnnpdf/wrapper/CMakeFiles/_nnpdf.dir/nnpdfPYTHON_wrap.cxx:5006:12:   required from 'static int swig::traits_asptr_stdseq<Seq, T>::asptr(PyObject*, swig::traits_asptr_stdseq<Seq, T>::sequence**) [with Seq = std::vector<NNPDF::DataSet>; T = NNPDF::DataSet; PyObject = _object; swig::traits_asptr_stdseq<Seq, T>::sequence = std::vector<NNPDF::DataSet>]'
/home/tommy/physics/nnpdfgit/nnpdf/bld/libnnpdf/wrapper/CMakeFiles/_nnpdf.dir/nnpdfPYTHON_wrap.cxx:5060:52:   required from 'static int swig::traits_asptr<std::vector<T> >::asptr(PyObject*, std::vector<T>**) [with T = NNPDF::DataSet; PyObject = _object]'
/home/tommy/physics/nnpdfgit/nnpdf/bld/libnnpdf/wrapper/CMakeFiles/_nnpdf.dir/nnpdfPYTHON_wrap.cxx:3854:37:   required from 'int swig::asptr(PyObject*, Type**) [with Type = std::vector<NNPDF::DataSet>; PyObject = _object]'
/home/tommy/physics/nnpdfgit/nnpdf/bld/libnnpdf/wrapper/CMakeFiles/_nnpdf.dir/nnpdfPYTHON_wrap.cxx:40600:34:   required from here
/home/tommy/physics/nnpdfgit/nnpdf/bld/libnnpdf/wrapper/CMakeFiles/_nnpdf.dir/nnpdfPYTHON_wrap.cxx:3937:8: warning: 'void* memset(void*, int, size_t)' clearing an object of type 'class NNPDF::DataSet' with no trivial copy-assignment; use assignment instead [-Wclass-memaccess]
 3937 |  memset(v_def,0,sizeof(Type));
      |  ~~~~~~^~~~~~~~~~~~~~~~~~~~~~
In file included from /home/tommy/physics/nnpdfgit/nnpdf/bld/libnnpdf/wrapper/CMakeFiles/_nnpdf.dir/nnpdfPYTHON_wrap.cxx:3192:
/home/tommy/physics/nnpdfgit/nnpdf/libnnpdf/src/NNPDF/dataset.h:29:9: note: 'class NNPDF::DataSet' declared here
   29 |   class DataSet : public CommonData, public FKSet
      |         ^~~~~~~
/home/tommy/physics/nnpdfgit/nnpdf/bld/libnnpdf/wrapper/CMakeFiles/_nnpdf.dir/nnpdfPYTHON_wrap.cxx: In instantiation of 'static Type swig::traits_as<Type, swig::pointer_category>::as(PyObject*, bool) [with Type = NNPDF::ThPredictions; PyObject = _object]':
/home/tommy/physics/nnpdfgit/nnpdf/bld/libnnpdf/wrapper/CMakeFiles/_nnpdf.dir/nnpdfPYTHON_wrap.cxx:3962:64:   required from 'Type swig::as(PyObject*, bool) [with Type = NNPDF::ThPredictions; PyObject = _object]'
/home/tommy/physics/nnpdfgit/nnpdf/bld/libnnpdf/wrapper/CMakeFiles/_nnpdf.dir/nnpdfPYTHON_wrap.cxx:4525:20:   required from 'swig::SwigPySequence_Ref<T>::operator T() const [with T = NNPDF::ThPredictions]'
/home/tommy/physics/nnpdfgit/nnpdf/bld/libnnpdf/wrapper/CMakeFiles/_nnpdf.dir/nnpdfPYTHON_wrap.cxx:4984:30:   required from 'void swig::assign(const SwigPySeq&, Seq*) [with SwigPySeq = swig::SwigPySequence_Cont<NNPDF::ThPredictions>; Seq = std::vector<NNPDF::ThPredictions>]'
/home/tommy/physics/nnpdfgit/nnpdf/bld/libnnpdf/wrapper/CMakeFiles/_nnpdf.dir/nnpdfPYTHON_wrap.cxx:5006:12:   required from 'static int swig::traits_asptr_stdseq<Seq, T>::asptr(PyObject*, swig::traits_asptr_stdseq<Seq, T>::sequence**) [with Seq = std::vector<NNPDF::ThPredictions>; T = NNPDF::ThPredictions; PyObject = _object; swig::traits_asptr_stdseq<Seq, T>::sequence = std::vector<NNPDF::ThPredictions>]'
/home/tommy/physics/nnpdfgit/nnpdf/bld/libnnpdf/wrapper/CMakeFiles/_nnpdf.dir/nnpdfPYTHON_wrap.cxx:5060:52:   required from 'static int swig::traits_asptr<std::vector<T> >::asptr(PyObject*, std::vector<T>**) [with T = NNPDF::ThPredictions; PyObject = _object]'
/home/tommy/physics/nnpdfgit/nnpdf/bld/libnnpdf/wrapper/CMakeFiles/_nnpdf.dir/nnpdfPYTHON_wrap.cxx:3854:37:   required from 'int swig::asptr(PyObject*, Type**) [with Type = std::vector<NNPDF::ThPredictions>; PyObject = _object]'
/home/tommy/physics/nnpdfgit/nnpdf/bld/libnnpdf/wrapper/CMakeFiles/_nnpdf.dir/nnpdfPYTHON_wrap.cxx:43915:34:   required from here
/home/tommy/physics/nnpdfgit/nnpdf/bld/libnnpdf/wrapper/CMakeFiles/_nnpdf.dir/nnpdfPYTHON_wrap.cxx:3937:8: warning: 'void* memset(void*, int, size_t)' clearing an object of type 'class NNPDF::ThPredictions' with no trivial copy-assignment; use assignment instead [-Wclass-memaccess]
 3937 |  memset(v_def,0,sizeof(Type));
      |  ~~~~~~^~~~~~~~~~~~~~~~~~~~~~
In file included from /home/tommy/physics/nnpdfgit/nnpdf/bld/libnnpdf/wrapper/CMakeFiles/_nnpdf.dir/nnpdfPYTHON_wrap.cxx:3194:
/home/tommy/physics/nnpdfgit/nnpdf/libnnpdf/src/NNPDF/thpredictions.h:23:11: note: 'class NNPDF::ThPredictions' declared here
   23 |     class ThPredictions {
      |           ^~~~~~~~~~~~~
/home/tommy/physics/nnpdfgit/nnpdf/bld/libnnpdf/wrapper/CMakeFiles/_nnpdf.dir/nnpdfPYTHON_wrap.cxx: In instantiation of 'static Type swig::traits_as<Type, swig::pointer_category>::as(PyObject*, bool) [with Type = std::pair<std::__cxx11::basic_string<char>, std::vector<std::__cxx11::basic_string<char> > >; PyObject = _object]':
/home/tommy/physics/nnpdfgit/nnpdf/bld/libnnpdf/wrapper/CMakeFiles/_nnpdf.dir/nnpdfPYTHON_wrap.cxx:3962:64:   required from 'Type swig::as(PyObject*, bool) [with Type = std::pair<std::__cxx11::basic_string<char>, std::vector<std::__cxx11::basic_string<char> > >; PyObject = _object]'
/home/tommy/physics/nnpdfgit/nnpdf/bld/libnnpdf/wrapper/CMakeFiles/_nnpdf.dir/nnpdfPYTHON_wrap.cxx:4525:20:   required from 'swig::SwigPySequence_Ref<T>::operator T() const [with T = std::pair<std::__cxx11::basic_string<char>, std::vector<std::__cxx11::basic_string<char> > >]'
/home/tommy/physics/nnpdfgit/nnpdf/bld/libnnpdf/wrapper/CMakeFiles/_nnpdf.dir/nnpdfPYTHON_wrap.cxx:4585:14:   required from 'swig::SwigPySequence_ArrowProxy<T> swig::SwigPySequence_InputIterator<T, Reference>::operator->() const [with T = std::pair<std::__cxx11::basic_string<char>, std::vector<std::__cxx11::basic_string<char> > >; Reference = const swig::SwigPySequence_Ref<std::pair<std::__cxx11::basic_string<char>, std::vector<std::__cxx11::basic_string<char> > > >]'
/home/tommy/physics/nnpdfgit/nnpdf/bld/libnnpdf/wrapper/CMakeFiles/_nnpdf.dir/nnpdfPYTHON_wrap.cxx:5724:25:   required from 'void swig::assign(const SwigPySeq&, std::map<K, T, Compare, Alloc>*) [with SwigPySeq = swig::SwigPySequence_Cont<std::pair<std::__cxx11::basic_string<char>, std::vector<std::__cxx11::basic_string<char> > > >; K = std::__cxx11::basic_string<char>; T = std::vector<std::__cxx11::basic_string<char> >; Compare = std::less<std::__cxx11::basic_string<char> >; Alloc = std::allocator<std::pair<const std::__cxx11::basic_string<char>, std::vector<std::__cxx11::basic_string<char> > > >]'
/home/tommy/physics/nnpdfgit/nnpdf/bld/libnnpdf/wrapper/CMakeFiles/_nnpdf.dir/nnpdfPYTHON_wrap.cxx:5006:12:   required from 'static int swig::traits_asptr_stdseq<Seq, T>::asptr(PyObject*, swig::traits_asptr_stdseq<Seq, T>::sequence**) [with Seq = std::map<std::__cxx11::basic_string<char>, std::vector<std::__cxx11::basic_string<char> > >; T = std::pair<std::__cxx11::basic_string<char>, std::vector<std::__cxx11::basic_string<char> > >; PyObject = _object; swig::traits_asptr_stdseq<Seq, T>::sequence = std::map<std::__cxx11::basic_string<char>, std::vector<std::__cxx11::basic_string<char> > >]'
/home/tommy/physics/nnpdfgit/nnpdf/bld/libnnpdf/wrapper/CMakeFiles/_nnpdf.dir/nnpdfPYTHON_wrap.cxx:5740:64:   required from 'static int swig::traits_asptr<std::map<_Key, _Tp, _Compare, _Alloc> >::asptr(PyObject*, swig::traits_asptr<std::map<_Key, _Tp, _Compare, _Alloc> >::map_type**) [with K = std::__cxx11::basic_string<char>; T = std::vector<std::__cxx11::basic_string<char> >; Compare = std::less<std::__cxx11::basic_string<char> >; Alloc = std::allocator<std::pair<const std::__cxx11::basic_string<char>, std::vector<std::__cxx11::basic_string<char> > > >; PyObject = _object; swig::traits_asptr<std::map<_Key, _Tp, _Compare, _Alloc> >::map_type = std::map<std::__cxx11::basic_string<char>, std::vector<std::__cxx11::basic_string<char> > >]'
/home/tommy/physics/nnpdfgit/nnpdf/bld/libnnpdf/wrapper/CMakeFiles/_nnpdf.dir/nnpdfPYTHON_wrap.cxx:3854:37:   required from 'int swig::asptr(PyObject*, Type**) [with Type = std::map<std::__cxx11::basic_string<char>, std::vector<std::__cxx11::basic_string<char> > >; PyObject = _object]'
/home/tommy/physics/nnpdfgit/nnpdf/bld/libnnpdf/wrapper/CMakeFiles/_nnpdf.dir/nnpdfPYTHON_wrap.cxx:24404:254:   required from here
/home/tommy/physics/nnpdfgit/nnpdf/bld/libnnpdf/wrapper/CMakeFiles/_nnpdf.dir/nnpdfPYTHON_wrap.cxx:3937:8: warning: 'void* memset(void*, int, size_t)' clearing an object of type 'struct std::pair<std::__cxx11::basic_string<char>, std::vector<std::__cxx11::basic_string<char> > >' with no trivial copy-assignment; use assignment instead [-Wclass-memaccess]
 3937 |  memset(v_def,0,sizeof(Type));
      |  ~~~~~~^~~~~~~~~~~~~~~~~~~~~~
In file included from /home/tommy/miniconda3/envs/nnpdf-test/x86_64-conda-linux-gnu/include/c++/9.3.0/bits/stl_algobase.h:64,
                 from /home/tommy/miniconda3/envs/nnpdf-test/x86_64-conda-linux-gnu/include/c++/9.3.0/bits/char_traits.h:39,
                 from /home/tommy/miniconda3/envs/nnpdf-test/x86_64-conda-linux-gnu/include/c++/9.3.0/string:40,
                 from /home/tommy/miniconda3/envs/nnpdf-test/x86_64-conda-linux-gnu/include/c++/9.3.0/stdexcept:39,
                 from /home/tommy/physics/nnpdfgit/nnpdf/bld/libnnpdf/wrapper/CMakeFiles/_nnpdf.dir/nnpdfPYTHON_wrap.cxx:3110:
/home/tommy/miniconda3/envs/nnpdf-test/x86_64-conda-linux-gnu/include/c++/9.3.0/bits/stl_pair.h:208:12: note: 'struct std::pair<std::__cxx11::basic_string<char>, std::vector<std::__cxx11::basic_string<char> > >' declared here
  208 |     struct pair
      |            ^~~~
[ 51%] Linking CXX shared module _nnpdf.so
[ 51%] Built target _nnpdf
Scanning dependencies of target common
[ 53%] Building CXX object nnpdfcpp/src/CMakeFiles/common.dir/common/src/nnpdfsettings.cc.o
[ 55%] Building CXX object nnpdfcpp/src/CMakeFiles/common.dir/common/src/md5.cc.o
[ 57%] Building CXX object nnpdfcpp/src/CMakeFiles/common.dir/common/src/loadutils.cc.o
[ 60%] Building CXX object nnpdfcpp/src/CMakeFiles/common.dir/common/src/datautils.cc.o
[ 62%] Building CXX object nnpdfcpp/src/CMakeFiles/common.dir/common/src/exportgrid.cc.o
[ 64%] Linking CXX static library libcommon.a
[ 64%] Built target common
Scanning dependencies of target nnfit
[ 66%] Building CXX object nnpdfcpp/src/CMakeFiles/nnfit.dir/nnfit/src/nnfit.cc.o
[ 68%] Building CXX object nnpdfcpp/src/CMakeFiles/nnfit.dir/nnfit/src/evolgrid.cc.o
[ 71%] Building CXX object nnpdfcpp/src/CMakeFiles/nnfit.dir/nnfit/src/fastaddchi2.cc.o
[ 73%] Building CXX object nnpdfcpp/src/CMakeFiles/nnfit.dir/nnfit/src/fitbases.cc.o
[ 75%] Building CXX object nnpdfcpp/src/CMakeFiles/nnfit.dir/nnfit/src/fitpdfset.cc.o
[ 77%] Building CXX object nnpdfcpp/src/CMakeFiles/nnfit.dir/nnfit/src/minimizer.cc.o
[ 80%] Building CXX object nnpdfcpp/src/CMakeFiles/nnfit.dir/nnfit/src/pdfbasis.cc.o
[ 82%] Building CXX object nnpdfcpp/src/CMakeFiles/nnfit.dir/nnfit/src/stopping.cc.o
[ 84%] Linking CXX executable ../../binaries/nnfit
/home/tommy/miniconda3/envs/nnpdf-test/bin/../lib/gcc/x86_64-conda-linux-gnu/9.3.0/../../../../x86_64-conda-linux-gnu/bin/ld: /home/tommy/miniconda3/envs/nnpdf-test/lib/libstdc++.so: undefined reference to `aligned_alloc@GLIBC_2.16'
/home/tommy/miniconda3/envs/nnpdf-test/bin/../lib/gcc/x86_64-conda-linux-gnu/9.3.0/../../../../x86_64-conda-linux-gnu/bin/ld: /home/tommy/miniconda3/envs/nnpdf-test/lib/libLHAPDF.so: undefined reference to `memcpy@GLIBC_2.14'
/home/tommy/miniconda3/envs/nnpdf-test/bin/../lib/gcc/x86_64-conda-linux-gnu/9.3.0/../../../../x86_64-conda-linux-gnu/bin/ld: /home/tommy/miniconda3/envs/nnpdf-test/lib/libstdc++.so: undefined reference to `clock_gettime@GLIBC_2.17'
/home/tommy/miniconda3/envs/nnpdf-test/bin/../lib/gcc/x86_64-conda-linux-gnu/9.3.0/../../../../x86_64-conda-linux-gnu/bin/ld: /home/tommy/miniconda3/envs/nnpdf-test/lib/libgfortran.so.4: undefined reference to `secure_getenv@GLIBC_2.17'
collect2: error: ld returned 1 exit status
nnpdfcpp/src/CMakeFiles/nnfit.dir/build.make:209: recipe for target 'binaries/nnfit' failed
make[2]: *** [binaries/nnfit] Error 1
CMakeFiles/Makefile2:280: recipe for target 'nnpdfcpp/src/CMakeFiles/nnfit.dir/all' failed
make[1]: *** [nnpdfcpp/src/CMakeFiles/nnfit.dir/all] Error 2
Makefile:148: recipe for target 'all' failed
make: *** [all] Error 2

```

--- a/doc/sphinx/source/get-started/installation.rst
+++ b/doc/sphinx/source/get-started/installation.rst
@@ -171,7 +171,14 @@ explained above, if the user has not already done so.
 
    .. code::
 
-       conda install pkg-config swig=3.0.10 cmake
+       conda install pkg-config swig cmake
+      
+   When working on a Linux system it is `currently 
+   <https://github.com/NNPDF/nnpdf/pull/1280>`_ also needed to run
+   
+   .. code::
+   
+       conda install sysroot_linux-64
 
 5. We now need to make the installation prefix point to our
    ``nnpdf-dev`` environment. Fortunately, when you activate the environment,

--- a/doc/sphinx/source/get-started/installation.rst
+++ b/doc/sphinx/source/get-started/installation.rst
@@ -178,7 +178,7 @@ explained above, if the user has not already done so.
    
    .. code::
    
-       conda install sysroot_linux-64
+       conda install sysroot_linux-64=2.17
 
 5. We now need to make the installation prefix point to our
    ``nnpdf-dev`` environment. Fortunately, when you activate the environment,

